src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonerResourceTest.kt

```diff
@@ -15,16 +15,40 @@ class PrisonerResourceTest : ResourceTest() {
   @Nested
   @DisplayName("GET /api/prisoners")
   inner class GetPrisoners {
+
     @Test
-    fun testCanFindSinglePrisoner() {
-      val httpEntity = createEmptyHttpEntity(GLOBAL_SEARCH)
-      val response = testRestTemplate.exchange(
-        "/api/prisoners?offenderNo=A1234AC",
-        HttpMethod.GET,
-        httpEntity,
-        object : ParameterizedTypeReference<String?>() {},
-      )
-      assertThatJsonFileAndStatus(response, HttpStatus.OK, "prisoners_single.json")
+    fun `returns 401 without an auth token`() {
+      webTestClient.get().uri("/api/prisoners?offenderNo=A1234AC")
+        .exchange()
+        .expectStatus().isUnauthorized
+    }
+
+    @Test
+    fun `returns 403 when there is no override role`() {
+      webTestClient.get().uri("/api/prisoners?offenderNo=A1234AC")
+        .headers(setClientAuthorisation(listOf()))
+        .exchange()
+        .expectStatus().isForbidden
+    }
+
+    @Test
+    fun `returns success if has override ROLE_GLOBAL_SEARCH`() {
+      webTestClient.get().uri("/api/prisoners?offenderNo=A1234AC")
+        .headers(setClientAuthorisation(listOf("ROLE_GLOBAL_SEARCH")))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("[0].offenderNo").isEqualTo("A1234AC")
+    }
+
+    @Test
+    fun `returns success if has override SYSTEM_USER`() {
```

Comment:
sorry I'm a bit new to this area - hasn't SYSTEM_USER gone?

Reply:
Good spot - the first set of tests I accidentally added (got confused with a) /api/prisoners/{offenderNo} and b) /api/prisoners?offenderNo=
so I left them in - I haven't got round to removing SYSTEM_USER from a) yet  (a long and tedious process) - so left my test in

Reply:
think I got confused with the url too even after reading your explanation of the above!